config/helm/chart/default/templates/Common/crd/dynatrace-operator-crd.yaml

```diff
@@ -0,0 +1,4 @@
+{{- $platformIsSet := printf "%s" (required "Platform needs to be set to kubernetes, openshift, google" (include "dynatrace-operator.platformSet" .))}}
+{{ if eq (include "dynatrace-operator.partial" .) "false" }}
+  {{ .Files.Get "crd/dynatrace-operator-crd.yaml" }}
```

Comment:
I'm not sure this is the right approach. How  will this be handled in a release package?
Just putting the file contents here directly would be more explicit.